python/tests/unit/arcticdb/version_store/test_aggregation.py

```diff
@@ -24,6 +24,7 @@
 from hypothesis.extra.pandas import column, data_frames, range_indexes
 
 
+@pytest.mark.xfail(reason="Test fails on linux conda, should be fixed by issue #1035")
```

Comment:
The `xfail` should specify the platform right? eg we don't expect this to fail on PyPi builds.